src/components/Header/index.module.scss

```diff
@@ -19,7 +19,7 @@ $defaultHeaderHeight: 90px;
 .header {
   position: sticky;
   top: 0;
-  z-index: 1;
+  z-index: 2;
```

Comment:
I find that reverting these `z-index` changes also works.

My original intent was precisely control of layers. Because there are multiple layers on the page:

1. background
2. embellished
3. page content
4. header
5. modal dialog and popover contents

So I just thought that these five layers have different cascading orders and should have different z-indexes to make sure the pages render correct.